### PR TITLE
Final Sunset

### PR DESCRIPTION
**Pull request by Ninja-ioKarma**
This is [SC]Karma's Final Sunset texture pack. Gives a nice sunset based theme.

### CODE REVIEW

File: texturepacks/FinalSunset/_meta.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "name": "Final Sunset",
+    "author": "[SC]Karma",
+    "description": "This is [SC]Karma's Final Sunset texture pack. Gives a nice sunset based theme."
```
Comment (itzTheMeow) [inline texturepacks/FinalSunset/_meta.json:4]: remove the first sentence

**Review by itzTheMeow — CHANGES_REQUESTED** [review]

**Comment by itzTheMeow** [comment]
this is actually pretty cool

**Comment by Ninja-ioKarma** [comment]
@itzTheMeow I updated it 👍

File: texturepacks/FinalSunset/_meta.json (hunk 1 above)
Comment (Ninja-ioKarma) [reply texturepacks/FinalSunset/_meta.json]: Updated it 👍

**Pull request merged**